projects/cdk/directives/autofilled/autofilled.directive.ts

```diff
@@ -1,53 +1,24 @@
-import {Directive, ElementRef, Inject, Output} from '@angular/core';
-import {typedFromEvent} from '@taiga-ui/cdk/observables';
-import {TuiDirectiveStylesService} from '@taiga-ui/cdk/services';
-import {map, skip} from 'rxjs/operators';
+import {Directive, EventEmitter, HostBinding, HostListener, Output} from '@angular/core';
 
-const ON = 'tuiAutofillOn';
-const OFF = 'tuiAutofillOff';
-const STYLE = `
-@keyframes ${ON} {
-    from {
-        content: '1';
-    }
-
-    to {
-        content: '2';
-    }
-}
-
-@keyframes ${OFF} {
-    from {
-        content: '2';
-    }
-
-    to {
-        content: '1';
-    }
-}`;
-
-// @dynamic
 @Directive({
     selector: '[tuiAutofilledChange]',
-    host: {
-        class: 'tui-autofill',
-    },
+    host: {class: 'tui-autofill'},
 })
 export class TuiAutofilledDirective {
+    @HostBinding('class._autofilled')
```

Comment:
I think this doesn't do anything, we add this class on host in `PrimitiveTextfield` and styles rely on it being there, not here.

Reply:
```ts
<input (tuiAutofillChanges)="next.emit($event)">
```

But it would be great for us to add a class to the host to determine that it is autofilled.

`_autofilled` - system css class looks like `_focused`, etc